opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -453,9 +417,6 @@ public static WebAddress parseWebAddress(String webAddr) {
             });
         });
 
-        // An example of how to add a data type for option parsing
-        OptionParser.accept(WebAddress.class, Indexer::parseWebAddress);
```

Comment:
I wonder if this was meant to be used for -U ?

Reply:
Perhaps. The validation of -U elsewhere in `Indexer` is superior.

Reply:
Okay then.